tests/_data/config/spryk/spryks/SprykWithNotUniqueSpryks.yml

```diff
@@ -0,0 +1,16 @@
+spryk: wrapper
```

Comment:
Not sure if everyone will understand the file name. Not a blocker for me but I would try to find a more descriptive name. SprykWrapperWithSprykUsedMoreThanOnce.yml?